Functional tests in external failed when building image

Functional tests in external failed when building image

```
07:22:22       [exec] Step 15/18 : RUN wget https://sourceforge.net/projects/ant-contrib/files/ant-contrib/1.0b3/ant-contrib-1.0b3-bin.tar.gz &&     tar -zxvf ant-contrib-1.0b3-bin.tar.gz &&     cp ant-contrib/ant-contrib-1.0b3.jar ${ANT_HOME}/lib
07:22:22       [exec]  ---> Running in fe1b59afde5a
07:22:22       [exec] [91m--2019-03-06 06:22:22--  https://sourceforge.net/projects/ant-contrib/files/ant-contrib/1.0b3/ant-contrib-1.0b3-bin.tar.gz
07:22:22       [exec] [0m[91mResolving sourceforge.net (sourceforge.net)... [0m[91m216.105.38.13
07:22:23       [exec] Connecting to sourceforge.net (sourceforge.net)|216.105.38.13|:443... [0m[91mconnected.
07:22:23       [exec] [0m[91mUnable to establish SSL connection.
07:22:23       [exec] The command '/bin/sh -c wget https://sourceforge.net/projects/ant-contrib/files/ant-contrib/1.0b3/ant-contrib-1.0b3-bin.tar.gz &&     tar -zxvf ant-contrib-1.0b3-bin.tar.gz &&     cp ant-contrib/ant-contrib-1.0b3.jar ${ANT_HOME}/lib' returned a non-zero code: 4
07:22:23       [exec] [0m
07:22:23  
07:22:23  BUILD FAILED
07:22:23  /home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/TestConfig/scripts/build_test.xml:71: The following error occurred while executing this line:
07:22:23  /home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/thirdparty_containers/build.xml:31: The following error occurred while executing this line:
07:22:23  /home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/thirdparty_containers/functional-test/build.xml:36: The following error occurred while executing this line:
07:22:23  /home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/thirdparty_containers/functional-test/build.xml:24: exec returned: 4
```

https://ci.adoptopenjdk.net/view/Test_external/job/openjdk8_j9_externaltest_x86-64_linux/361/console
https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_j9_externaltest_x86-64_linux/150/console
https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_j9_externaltest_extended_x86-64_linux/139/console
https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_hs_externaltest_x86-64_linux/160/console
https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_hs_externaltest_extended_x86-64_linux/151/console

--- a/thirdparty_containers/functional-test/dockerfile/Dockerfile
+++ b/thirdparty_containers/functional-test/dockerfile/Dockerfile
@@ -33,6 +33,9 @@ FROM $IMAGE_NAME:$IMAGE_VERSION
 RUN apt-get update \
 	&& apt-get -y install \
 	ant \
+	ant-contrib \
+	autoconf \
+	bash \
 	apt-transport-https \
 	ca-certificates \
 	dirmngr \
@@ -41,20 +44,12 @@ RUN apt-get update \
 	make \
 	unzip \
 	vim \
+	zip \
 	wget \
-	gcc
-
-RUN apt-get -y install \
+	gcc \
 	libtext-csv-perl \
 	libjson-perl \
 	libxml-parser-perl
-
-# Install ant
-ENV ANT_VERSION 1.10.5
-RUN wget --no-check-certificate https://www.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.tar.gz && \
-    gunzip apache-ant-${ANT_VERSION}-bin.tar.gz && \
-    tar -xvf apache-ant-${ANT_VERSION}-bin.tar
-
 
 VOLUME ["/java"]
 ENV JAVA_HOME=/java \
@@ -68,12 +63,6 @@ COPY ./dockerfile/functional-test.sh /functional-test.sh
 # Clone the repo where the functional tests reside. 
 WORKDIR /
 RUN pwd
-
-ENV ANT_HOME ${WORKDIR}/apache-ant-${ANT_VERSION}
-
-RUN wget https://sourceforge.net/projects/ant-contrib/files/ant-contrib/1.0b3/ant-contrib-1.0b3-bin.tar.gz && \
-    tar -zxvf ant-contrib-1.0b3-bin.tar.gz && \
-    cp ant-contrib/ant-contrib-1.0b3.jar ${ANT_HOME}/lib
 
 # Clone OpenJ9 functional test repo
 RUN git clone https://github.com/eclipse/openj9.git

